matplotlib2tikz/path.py

```diff
@@ -129,30 +133,36 @@ def draw_pathcollection(data, obj):
         else:
             draw_options.append('colormap/' + mycolormap)
 
-    if len(obj.get_sizes()) == len(dd):
-        # See Pgfplots manual, chapter 4.25.
-        # In Pgfplots, \mark size specifies raddi, in matplotlib circle areas.
-        radii = numpy.sqrt(obj.get_sizes() / numpy.pi)
-        dd = numpy.column_stack([dd, radii])
-        labels.append('sizedata' + 14*' ')
-        draw_options.extend([
-            'visualization depends on='
-            '{\\thisrow{sizedata} \\as\\perpointmarksize}',
-            'scatter/@pre marker code/.append style='
-            '{/tikz/mark size=\\perpointmarksize}',
-            ])
-
-    do = ' [{}]'.format(', '.join(draw_options)) if draw_options else ''
-    content.append('\\addplot{}\n'.format(do))
-
-    to = ' [{}]'.format(', '.join(table_options)) if table_options else ''
-    content.append('table{}{{%\n'.format(to))
-
-    content.append((' '.join(labels)).strip() + '\n')
-    fmt = (' '.join(dd.shape[1] * ['%+.15e'])) + '\n'
-    for d in dd:
-        content.append(fmt % tuple(d))
-    content.append('};\n')
+    num_paths = len(obj.get_paths()) if is_contour else 1
+
+    for i in range(num_paths):
```

Comment:
Why can we not just do
```python
for path in ob.get_paths():
```
here?

Reply:
Tests seems to pass with the change above. And it's more Python-like.